.github/workflows/update-docker-images.yml

```diff
@@ -138,17 +138,9 @@ jobs:
             platforms: linux/arm,linux/arm64,linux/amd64,linux/ppc64le,linux/s390x
             needs-updating: ${{ needs.check.outputs.needs-updating-debian }}
           - image: alpine
-            marker: vsr
```

Comment:
we're loosing a vsr marker? can we add it to any of the images?

Reply:
Ah good call, I've added them back in other images 👍